package t.TestC;

public class Main {

	public static void main(String[] args) throws RuntimeException {
		try {
			System.out.println("!!!!!!!!!!!!");
			System.out.println("!!Started!!!");
			System.out.println("!!!!!!!!!!!!");
			Test.testCs();

		} catch (Throwable e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		System.out.println("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
		System.out.println("!!!!!!!!!!!!!!FINISH!!!!!!!!!!!!!!!!!!!");
		System.out.println("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");

	}

}
